Do not fail if related usns fail (SC-1300) (#2497)
canonical/ubuntu-pro-client: docs/references/what_is_a_usn.md

> # What is a USN ?

Reviewer: ```suggestion
# What is a USN?
```

> A USN (Ubuntu Security Notice) is the way for Canonical to public catalog and display security

Reviewer: ```suggestion
An Ubuntu Security Notice (USN) is the way that Canonical publicly catalogues and displays security
```

Reviewer: I think we ought to also include this link https://ubuntu.com/security/notices

Author: Good point

> vulneratibility for Ubuntu packages. Usually, a USN is composed by one or more

Reviewer: ```suggestion
vulnerabilities for Ubuntu packages. Usually, a USN is composed of one or more
```

[CVEs](what_is_a_cve.md) and it also contains update instructions to fix the issue, if a fix is
already available.

> Similar to a CVE, a USN also follows a name pattern like this one [USN-5963-1](https://ubuntu.com/security/notices/USN-5963-1)

Reviewer: ```suggestion
USNs follow a naming convention of the format: [USN-5963-1](https://ubuntu.com/security/notices/USN-5963-1)
```
